iosApp/iosAppTests/Support/PublicAPI/PresenceApi.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PresenceApi.swift
+//  MessengerUITests
+//
+//  Created by Morehouse, Matthew on 6/3/21.
+//  Copyright © 2021 Genesys. All rights reserved.
+//
+
+import XCTest
+
+extension ApiHelper {
```

Comment:
Do we need PresenceAPI? I am not aware that transport has any means of supporting that feature. At least, no in observable universe.

Reply:
Presence API lets us set agents to be On Queue and will let us respond to messages that are sent from the transport SDK. So as far as the sdk goes, no the presence API is not needed. But testing needs to be able to set agents to on queue to make sure everything is working as expected.